virtual_rainforest/core/data.py

```diff
@@ -348,7 +348,7 @@ def save_to_netcdf(self, output_file_path: Path) -> None:
         # If the file path is okay then write the model state out as a NetCDF
         self.data.to_netcdf(output_file_path)
 
-    def add_from_dict(self, output_dict: Dict[str, DataArray]) -> None:
+    def add_from_dict(self, output_dict: dict[str, DataArray]) -> None:
```

Comment:
What is the difference between `dict` and `Dict `(just out of curiosity)?

Reply:
Their behaviour is identical. `Dict` was originally defined in `typing` as a type hint for a dictionary, but python 3.9 was changed to allow collections (e.g. `dict`) to be used to directly give type hints. So older versions of python have to import `Dict` from `typing` to type hint, but in more recent versions it's considered to be deprecated as you can just use `dict` instead without importing